### PR TITLE
Citations and documentation update

### PR DESCRIPTION
**Pull request by elizabethmcd**
This PR updates the citations, documentation, and MultiQC report output with additions for `samtools stats`

### CODE REVIEW

File: assets/methods_description_template.yml [hunk 1]
```diff
@@ -3,3 +3,23 @@ description: "Suggested text and references to use when describing pipeline usag
...
 section_href: "https://github.com/Arcadia-Science/hifi2genome"
 plot_type: "html"
+data: |
+  <h4>Methods</h4>
+  <p> Data was processed using Arcadia-Science/hifi2genome v${workflow.manifest.version} ${doi_text}.
+  <p> The pipeline was executed with Nextflow v${workflow.nextflow.version} (<a href="https://doi.org/10.1038/nbt.3820">[3]</a>) with the following command:</p>
+  <pre><code>${workflow.commandLine}</code></pre>
+  <h4>References</h4>
+  <ul>
+    <li> [INSERT REF]
```
Comment (taylorreiter) [inline assets/methods_description_template.yml:13]: ?

File: assets/methods_description_template.yml [hunk 2]
```diff
@@ -3,3 +3,23 @@ description: "Suggested text and references to use when describing pipeline usag
...
+    <li>[2] Ewels, PA, Peltzer, A, Fillinger, S, Patel, H, Alneberg, J, Wilm, A, Garcia, MU, Di Tommaso, P, Nahnsen, S. (2020). The nf-core framework for community-curated bioinformatics pipelines. <a href="https://doi.org/10.1038/s41587-020-0439-x">https://doi.org/10.1038/s41587-020-0439-x</a></li>
+    <li>[3] Di Tommaso, P, Chatzou, M, Floden, EW, Barja, PP, Palumbo, E, Notredame, C. (2017). Nextflow enables reproducible computational workflows. <a href="https://doi.org/10.1038/nbt.3820">https://doi.org/10.1038/nbt.3820</a></li>
+
+  </ul>
+  <div class="alert alert-info">
+    <h5>Notes:</h5>
+    <ul>
+      ${nodoi_text}
+      <li>The command above does not include parameters contained in any configs or profiles that may have been used. Ensure the config file is also uploaded with your publication!</li>
```
Comment (taylorreiter) [inline assets/methods_description_template.yml:22]: huh interesting. i don't hate it. kind of like "STAR methods"

File: assets/methods_description_template.yml (hunk 1 above)
Comment (taylorreiter) [reply assets/methods_description_template.yml:13]: oh i get it, as a placeholder for the hifi2genome pub?

**Review by taylorreiter — APPROVED** [review]
lgtm :) you may also want to update the citations section of the README

File: assets/methods_description_template.yml (hunk 1 above)
Comment (elizabethmcd) [reply assets/methods_description_template.yml:13]: yes!

File: assets/methods_description_template.yml (hunk 2 above)
Comment (elizabethmcd) [reply assets/methods_description_template.yml:22]: I think I just mostly took what was in the seqqc methods description but maybe I was looking at the wrong thing...

**Comment by elizabethmcd** [comment]
updated the citation format in the readme as well

**Pull request merged**